### ISSUE TITLE
SIGABRT on iOS device (non-simulated): dyld: Library not loaded: @rpath/Deque.framework/Deque

### ISSUE THREAD

**Issue by bgromov** [tgu/RosSwift#2]
### Expected behavior

The app is expected to run on real device.

### Actual behavior

It doesn't.

The app crashes on startup with `SIGABRT` on real iOS device (iPhone 11, 13.5.1) with the following message:

```
dyld: Library not loaded: @rpath/Deque.framework/Deque
  Referenced from: /private/var/containers/Bundle/Application/8F631FD7-6EDD-431B-B728-84C4B9A55893/RosSwift-Issue.app/RosSwift-Issue
  Reason: image not found
```

There are no problems in simulator.

### Steps to reproduce

1. Create new Single View App project for iOS in Xcode (11.6 or 12.0 beta 3):
   - Language: Swift
   - User Interface: SwiftUI
2. Add `RosSwift` package in Project Settings -> Swift Packages
3. Run on iPhone device

### If possible, minimal yet complete reproducer code (or URL to code)

No need to do any modifications to a generated project.

### RosSwift version/commit hash

RosSwift v1.2.0

### Swift & OS version (output of `swift --version && uname -a`)
```
$ swift --version && uname -a
Apple Swift version 5.2.4 (swiftlang-1103.0.32.9 clang-1103.0.32.53)
Target: x86_64-apple-darwin19.5.0

Darwin macbook-pro 19.5.0 Darwin Kernel Version 19.5.0: Tue May 26 20:41:44 PDT 2020; root:xnu-6153.121.2~2/RELEASE_X86_64 x86_64
```

---

I remember to have no issues a few months back and tried to downgrade to a previous `RosSwift` version (1.0.0) but it crashes anyway.

I also tried to add `SipHash` package (contains `Deque`) directly to the project, still no luck.

Any ideas what could be wrong?

Thanks.

**Comment by bgromov** [comment]
> I also tried to add `SipHash` package (contains `Deque`) directly to the project, still no luck.

That's obviously wrong. There was a copy-paste typo in `Deque` readme and I didn't think enough before checking the URL and the package name. Sorry.

Adding `Deque` directly to the project actually _**does**_ solve the problem. It well could be just an Xcode bug, so please fill free to close the issue.

Thanks for the RosSwift!

**Comment by tgu** [comment]
Seems to be fixed in IOS 13.6,

**Comment by tgu** [comment]
Thanks for pointing out this bug, changes in security policys for iOS apps seems to disallow dynamic libraries. Changing Deque to a static library seems to fix the problem.
